lib/node_modules/@stdlib/assert/is-duration-string/docs/repl.txt

```diff
@@ -0,0 +1,47 @@
+
+{{alias}}( value )
+    Tests if a value is a duration string.
+
+    The function validates that a value is a string. For all other types,
```

Comment:
@Planeshifter The line wrapping looks off here. Either this line has a bit more room, or subsequent lines are too long.

Reply:
My guess is that this stems from removing the backticks in a prior revision.

Reply:
Good catch! Fixed.